src/applications/personalization/profile360/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -6,12 +6,10 @@ import * as paymentInformationActions from '../../actions/paymentInformation';
 let oldFetch;
 let oldGA;
 
-const setup = () => {
+const setup = ({ mockGA }) => {
```

Comment:
I realize we could easily just pass a bool directly to `setup` rather than this overengineered solution of passing in an object with a `mockGA` key, _but_ I think having the "named argument" makes calling `setup()` so much easier to understand on lines 47 and 83